src/main/java/com/minecolonies/coremod/client/gui/WindowHutBuilder.java

```diff
@@ -76,21 +135,57 @@ public int getElementCount()
             {
                 return resources.size();
             }
-
             @Override
             public void updateElement(final int index, @NotNull final Pane rowPane)
             {
-                final Object obj = entries[index];
+                    final BuildingBuilder.BuildingBuilderResource resource = resources.get(index);
+                    Label resourceLabel = rowPane.findPaneOfTypeByID(RESOURCE_NAME, Label.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'resourceLabel' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)